tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -489,16 +481,58 @@ module.exports = {
     /**
      * @param {string} collaborator Name of the collaborator
      * @param {string} days number of days to be added or subtracted from current date
+     * @param {boolean} expectToSucceed
      *
      * @return {Promise<*>}
      */
     changeCollaboratorExpiryDate: async function (collaborator, days) {
       await collaboratorDialog.clickEditShare(collaborator)
-      await this.api.page
-        .FilesPageElement
-        .expirationDatePicker()
-        .setExpirationDate(days)
+      const dateToSet = sharingHelper.calculateDate(days)
+      const expectToSucceed = await this
```

Comment:
variable name does not make sense, how does `setExpirationDate` know what is expected?